cartographer_ros/configuration_files/backpack_2d.lua

```diff
@@ -23,6 +23,7 @@ options = {
   published_frame = "base_link",
   odom_frame = "odom",
   provide_odom_frame = true,
+  project_pose_to_2d_plane = false,
```

Comment:
Please have follow-up PRs ready for cartographer_fetch and cartographer_turtlebot.